pkg/ipamd/datastore/data_store.go

```diff
@@ -122,6 +122,19 @@ var (
 		},
 		[]string{"cidr"},
 	)
+	noAvailableIPAddrs = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "awscni_err_no_avail_addrs",
+			Help: "The number of pod IP assignments that fail due to no available IP addresses",
+		},
+	)
+	eniUtilization = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "awscni_eni_util",
+			Help: "The number of allocated ips partitioned by eni",
+		},
+		[]string{"fn"},
```

Comment:
nit: the string key should be renamed from "fn", which is short for "function", to "eni"